lib/Habitat.lua

```diff
@@ -11,9 +11,10 @@ local fs = import("./fs")
 local Habitat = {}
 Habitat.__index = Habitat
 
-function Habitat.new()
+function Habitat.new(flags)
```

Comment:
We should add a line to set `flags` to `{}` if it isn't passed, so that current use cases don't have to start worrying about flags.

Reply:
added!